test/flycheck-test.el

```diff
@@ -1911,7 +1911,7 @@ and extension, as in `file-name-base'."
                                              :filename "foo.cpp"))))
     (should
      (equal (flycheck-fold-include-levels errors "In file included from")
-            (list (flycheck-error-new-at 1 0 'error "In include foo.h"
+            (list (flycheck-error-new-at 1 0 'error "In include foo.h: b is not a member of hi"
```

Comment:
That's not actually correct.  If you look at the test errors above, you'll see that this message comes from `foo.cpp` and not from `foo.h`.

`in definition of macro CHECK` comes from `foo.h`.